dali/common.h

```diff
@@ -39,6 +39,9 @@ namespace dali {
 const float M_PI = 3.14159265358979323846;
 #endif
 
+const int MAX_ALLOWED_TRT_OP_INPUT = 16;
```

Comment:
These are TRT-specific, so shouldn't be in common files. They should be in `tensorRTInfer.h` instead

Reply:
I will update this